lints/lint_mp_modulus_must_be_divisible_by_8.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * ZLint Copyright 2019 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/********************************************************************
+Section 5.1 - Algorithms
+RSA keys whose modulus size in bits is divisible by 8, and is at least 2048.
+********************************************************************/
+
+package lints
+
+import (
+	"crypto/rsa"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type modulusDivisibleBy8 struct{}
+
+func (l *modulusDivisibleBy8) Initialize() error {
+	return nil
+}
+
+func (l *modulusDivisibleBy8) CheckApplies(c *x509.Certificate) bool {
+	if c.PublicKeyAlgorithm == x509.RSA {
```

Comment:
This can be removed

Reply:
This was due to an incorrect commit directly from the github ui, fixed it.